make/modules/java.base/Lib.gmk

```diff
@@ -233,3 +233,23 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
 
   TARGETS += $(BUILD_LIBFALLBACKLINKER)
 endif
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIB_X86_64, \
+      NAME := x86_64, \
```

Comment:
This looks like a rather generic name for a library. I would expect something a bit more descriptive. 

I also noted that @vnkozlov questioned needing a separate library for this and I didn't really find an answer. What do we gain from separating this into a separate dynamic library?

Reply:
Hi Erik, 
As per your suggestion, the name of the library is updated to `libx86_64_sort`. Thank you for fixing the indentation in the build script! 
Please see the changes in the latest commit that was pushed.